genienlp/predict.py

```diff
@@ -54,6 +55,8 @@
 
 logger = logging.getLogger(__name__)
 
+quoted_pattern_maybe_space = re.compile(r'\"\s?([^"]*?)\s?\"')
```

Comment:
This regular expression is not correct (it will match `"foo"` without space which is not a string parameter, most importantly it will match `""` which is also not a copiable parameter.